TestConsoleApp/Program.cs

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
deze file weghalen, is niet nodig in main
sla hem wel op want is handig voor testen